src/main/java/net/minecraftforge/event/entity/EndermanAttackEvent.java

```diff
@@ -0,0 +1,31 @@
+package net.minecraftforge.event.entity;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.monster.EndermanEntity;
+import net.minecraft.entity.monster.ZombieEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraftforge.eventbus.api.Cancelable;
+
+@net.minecraftforge.eventbus.api.Cancelable
```

Comment:
Why the FQN here?
And please add some documentation: What happens when the event is cancelled? When is this fired?